src/server/src/entities/student.ts

```diff
@@ -52,6 +53,15 @@ export class Student extends BaseEntity {
     @Column()
     userId: number;
 
+    @OneToOne(() => WorkExperience, {
```

Comment:
I think there should be a one-to-many relationship between the student and workExperien tables. "A student can have many work experiences but each work experience only belongs to one student"

Reply:
Oh, I see... I'll make the change.